sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentAnalysisAudience.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.AI.FormRecognizer.DocumentAnalysis
+{
+    /// <summary> Cloud audiences available for FormRecognizer.DocumentAnalysis. </summary>
+    public readonly partial struct DocumentAnalysisAudience : IEquatable<DocumentAnalysisAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DocumentAnalysisAudience"/> object.
+        /// </summary>
+        /// <param name="value">The Azure Active Directory audience to use when forming authorization scopes. For FormRecognizer.DocumentAnalysis, this value corresponds to a URL that identifies the Azure cloud where the resource is located. For more information: <see href="https://docs.microsoft.com/azure/azure-government/documentation-government-cognitiveservices" />.</param>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
```

Comment:
Should we add line for `ArgumentException` when `value` is empty?   (that's the behavior of `AssertNotNullOrEmpty`)